py3-dbs3-pycurl-client.spec

```diff
@@ -0,0 +1,38 @@
+### RPM cms py3-dbs3-pycurl-client 3.16.0
+## INITENV +PATH PYTHONPATH %i/${PYTHON_LIB_SITE_PACKAGES}
+## INITENV +PATH PYTHONPATH %i/x${PYTHON_LIB_SITE_PACKAGES}
+
+#%define webdoc_files %{installroot}/%{pkgrel}/doc/
+Source0: git://github.com/dmwm/DBS.git?obj=master/%{realversion}&export=DBS&output=/%{n}.tar.gz
+Requires: python3 py3-cjson py3-pycurl curl
```

Comment:
As you noticed, there is no cjson in python3, neither a py3-cjson spec file.

Reply:
fixed

Reply:
fixed